docs/en/02_Developer_Guides/09_Security/04_Secure_Coding.md

```diff
@@ -705,6 +705,15 @@ SilverStripe\Control\Session:
   cookie_secure: true
 ```
 
+The same treatment should be applied to the cookie responsible for remembering logins across sessions:
+
+```yml
+SilverStripe\Core\Injector\Injector:
```

Comment:
I think we should document how to do this only in Live mode or not on dev as I think that'll be quite a common usecase